Src/BootCamp.Chapter/Program.cs

```diff
@@ -23,10 +23,35 @@ static void Main(string[] args)
 
             Console.Write("Hello world!" + System.Environment.NewLine);
 
-            Console.WriteLine("Tom's age is " + tAge + "and his height is " + tHeight + " and his weight is " + tWeight + " BMI is: " + tBmi);
-            Console.WriteLine("Colby's age is " + cAge + "and his height is " + cHeight + " and his weight is " + cWeight + " BMI is: " + cBmi);
+            Console.WriteLine("Tom's age is " + tAge + " and his height is " + tHeight + " and his weight is " + tWeight + " BMI is: " + tBmi);
+            Console.WriteLine("Colby's age is " + cAge + " and his height is " + cHeight + " and his weight is " + cWeight + " BMI is: " + cBmi);
             Console.WriteLine("Blakes's age is " + bAge + " and his height is " + bHeight + " and his weight is " + bWeight + " BMI is: " + bBmi);
+
+            Console.WriteLine("Press any key to continue...");
+
+            Console.ReadLine();
+            Console.Clear();
+
+        Start:
+
+            Console.WriteLine("Please enter your name. ");
+            string yourName = Console.ReadLine();
+            Console.WriteLine("Hello " + yourName);
+
+            Console.WriteLine("Please enter your age now. ");
+            int yourAge = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Please enter your height now. ");
+            double yourHeight = double.Parse(Console.ReadLine());
+            Console.WriteLine("Please enter your weight now. ");
+            int yourWeight = Convert.ToInt32(Console.ReadLine());
+            double yourBMI = yourWeight / yourHeight;
+            Console.WriteLine("Alright " + yourName + ", " + "you are " + yourAge + " years old, and your BMI based off your provided measurements is " + yourBMI);
+
+            Console.WriteLine("Press any key to continue...");
+
             Console.ReadLine();
+            Console.Clear();
+            goto Start;
```

Comment:
Yipe! 

Consider a simple way of asking for two people's data without using `goto`. One way might be: 

    for (int i = 0; i < 2; i++)
    {
      // Ask and output here
    }

If you haven't seen a [for loop](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/for) before, that's fine! Basically, what this does is declare a variable `i`, makes sure that `i < 2`, then runs the code inside the brackets, then adds 1 to `i` and repeats. 

I definitely want to discourage to use of `goto`, in any circumstance.